examples/cmd/main.go

```diff
@@ -264,7 +264,16 @@ func (c *DownloaderCmd) commit(dp *DownloadersPool) error {
 }
 
 func (c *DownloaderCmd) buildDownloadersPool(logger log.Logger, db *sql.DB) (*DownloadersPool, error) {
-	var clients []*http.Client
+	var builder downloaderBuilder
```

Comment:
`downloaderBuilder` is defined in pool file but isn't used anywhere except inside this private function.
Most probably something like this will be even simpler (or at least shorter):
```go
	var downloaders []*github.Downloader
	for _, t := range c.Tokens {
		client := oauth2.NewClient(context.TODO(), oauth2.StaticTokenSource(
			&oauth2.Token{AccessToken: t},
		))
		if c.LogHTTP {
			setLogTransport(client, logger)
		}

		var d *github.Downloader
		if db == nil {
			d = github.NewStdoutDownloader(client)
		} else {
			d = github.NewDownloader(client, db)
		}


		downloaders = append(downloaders, d)
	}

	return NewDownloadersPool(downloaders)
```

Reply:
okay! it was just to avoid the check, but yup, I don't think that it's going to iterate millions of tokens...